feat: screenshot improvements

lidel opened the pull request:
This ensures the CID is properly passed to copied link, and that filename is preserved when a single screen setup is used (produces single screenshot). Switched to reliable gateway, as share.ipfs.io is not maintained.

We also avoid low level pinning  (MFS is enough) and ensure the produced CID is the same as on CLI (avoiding problems described in https://github.com/ipfs-shipyard/ipfs-webui/issues/676).

jessicaschilling reviewed (approved):
Confirmed this works for me on Mac, including generating two separate files (Screen 1.png and Screen 2.png) for my multi-screen setup.

One nit: the directory names aren't super user-friendly, probably out of scope for this issue but just wanted to note it.
![image](https://user-images.githubusercontent.com/1507828/95602233-ddfd0300-0a11-11eb-8a48-f8f5c53a0d5a.png)

lidel commented:
@jessicaschilling good catch. In  IPFS Companion we use `/ipfs-companion-imports/%Y-%M-%D_%h%m%s/` and numbers are padded with zeroes. Seconds are kinda needed if someone takes multiple screenshots one after another, just for sorting purposes. 

I've ported what we do in Companion as a starting point, let me know if anything should be tweaked:

> ![2020-10-09--18-10-16](https://user-images.githubusercontent.com/157609/95606680-f7766d00-0a5a-11eb-8b95-206496041b57.png)

- month was wrong (had to +1, because JS lol)
- added padding and removed spaces for better lexicographical sorting and better-looking share URL

jessicaschilling commented:
@lidel Thank you! This sounds good.
